tuning/sft_trainer.py

```diff
@@ -651,6 +655,25 @@ def main():
             )
             sys.exit(INTERNAL_ERROR_EXIT_CODE)
 
+    # post process lora
+    if isinstance(tune_config, peft_config.LoraConfig):
```

Comment:
can we do this with a flag 'vLLM_format_LoRA' ? I am not sure the formatted files will work with HF inference and the postprocessing shouldnt be needed for all use cases - which dont need vLLM for example